lucene/core/src/java/org/apache/lucene/index/DocumentsWriterDeleteQueue.java

```diff
@@ -573,6 +573,10 @@ long getMaxCompletedSeqNo() {
     }
   }
 
+  private static LongSupplier getPrevMaxSeqIdSupplier(AtomicLong nextSeqNo) {
```

Comment:
Maybe add a comment to this method referencing the issue, Simon?

Reply:
done